### ISSUE TITLE
VEDIT 1.36 VDSETCRT cannot find CRT.TBL; similar issue with 1.40

### ISSUE THREAD

**Issue by johnsonjh** [dmsc/emu2#68]
* VEDIT 1.36 VDSETCRT:
```
$ emu2 vdsetcrt.com vedit86.set vedit.com

   VDSETCRT  -  Version 1.36
*** CRT.TBL NOT FOUND, OR BAD FORMAT ***
```

The CRT.TBL file exists and this error does not occur in real DOS.

* VEDIT 1.40 INSTALL: 

A very similar issues happens with VEDIT 1.40’s INSTALL:

```
$ emu2 install.exe vplus86c.set vedit.com
INSTALL - Customization for VEDIT
Copyright (c) 1985 CompuView Products Incorporated.

Unable to open vplus86c.set
```

Do note that the keys are a bit different for 1.36 - you use ESC-ESC to terminate a command, and not enter.  That’s proper and not a bug in the emulator.

**Comment by johnsonjh** [comment]
(The old 1.36 86-DOS and 1.40 MS-DOS versions are in the https://github.com/johnsonjh/VEDIT repo, in the `oldbin` directory.)

**Comment by dmsc** [comment]
Hi!

The problem is that the application reserves 33 bytes for each FCB, but EMU2 expects (and clears) 36 bytes.

I will investigate if it is possible to not clear the "random access record number" on FCB open.

**Comment by dmsc** [comment]
Ok, after testing other programs, I removed the initialization of the random-access position in the FCB on open. I also implemented function 17h and fixes delete via FCB, now VEDIT 1.36 works.

VEDIT 1.40 still does not work, I will investigate further.

**Comment by dmsc** [comment]
Hi!

> * VEDIT 1.40 INSTALL:
> 
> A very similar issues happens with VEDIT 1.40’s INSTALL:
> 
> ```
> $ emu2 install.exe vplus86c.set vedit.com
> INSTALL - Customization for VEDIT
> Copyright (c) 1985 CompuView Products Incorporated.
> 
> Unable to open vplus86c.set
> ```

About vedit 1.40, there is no `vplus86c.set` file in the folder, the correct name is `vedit86c.set`, if using this filename, all works.

So, bug is closed now!
